src/component/textbox/Text.js

```diff
@@ -3,10 +3,9 @@ import Word from './Word';
 
 function Text(props) {
   function generateWords() {
-    let words = [];
-    for (var i = 0; i < props.text.length; i++) {
-      words[i] = renderWord(props.text[i], i);
-    }
+    let words = props.text.map((word, index) => {
+      return renderWord(word, index);
+    });
     return words;
   }
```

Comment:
Some syntactic sugar to make this a little cleaner.

```
function generateWords() => (
  props.text.map((word, index) => {
    return renderWord(word, index);
  }
)
```

I also wouldn't mind changing `props.text` to `props.words`; It's an array of words, not a text string.